### PR TITLE
WG selective routing: clarify why Step 9

### PR DESCRIPTION
**Pull request by paride**

### CODE REVIEW

**Comment by paride** [comment]
@Greelan I hope I figured this out correctly.

File: source/manual/how-tos/wireguard-selective-routing.rst [hunk 1]
```diff
@@ -196,11 +196,13 @@ The step has two parts - first creating a second Alias for all local (private) n
...
 
---------------------------
-Step 9 - Configure routing
---------------------------
+--------------------------------------------------------------
+Step 9 - Configure routing for traffic generated by the router
+--------------------------------------------------------------
 
-- Then go to :menuselection:`Firewall --> Rules --> Floating`
+Services running on the router and configured to use the VPN interface will generate traffic using the VPN address as the source IP. This traffic has to be routed to the VPN gateway. Note that locally generated traffic is not affected by NAT or by the firewall rule created in Step 8.
```
Comment (Greelan) [inline source/manual/how-tos/wireguard-selective-routing.rst:203]: Suggest this be changed slightly to:

`Services running on the router and configured to use the VPN interface must have their traffic routed to the VPN gateway in order to use the VPN. Note that locally generated traffic is not affected by NAT or by the firewall rule created in Step 8.`

**Comment by paride** [comment]
I almost forgot about creating this PR. @Monviech if the suggestion looks good to you, I can update my branch with it.

**Comment by Monviech** [comment]
Yes it looks okay to me. Lets update the branch and then I'll merge it. :)

**Comment by Monviech** [comment]
Thank you~

**Pull request merged**